src/ua/burkavtsov/hw22/EMH.java

```diff
@@ -0,0 +1,65 @@
+package ua.burkavtsov.hw22;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import javax.persistence.Persistence;
+
+public class EMH {
+
+    public static void main(String[] args) {
+        EntityManagerFactory emf = Persistence.createEntityManagerFactory("yourPersistenceUnitName");
+        EntityManager em = emf.createEntityManager();
```

Comment:
You can use try-with-resources here.

Also, in this example, no need to open transaction each time you do smth with DB. you can open it only once, do all you need and commit/rollback/